server/tests/accommodation.test.js

```diff
@@ -0,0 +1,91 @@
+import chaiHttp from 'chai-http';
+import chai, { expect } from 'chai';
+import app from '../index';
+import users from './mochData/users';
+import { validAccommodation, invalidAccommodation, existedAccommodation } from './mochData/accommodation';
+
+chai.use(chaiHttp);
+const router = () => chai.request(app);
+
+describe('Accomodation Test Suite', () => {
+  describe('accommodation creation with authorized user', () => {
+    let authorizedToken;
+
+    before((done) => {
+      router()
+        .post('/api/v1/auth/signin')
+        .send(users[6])
+        .end((err, res) => {
+          const { data } = res.body;
+          authorizedToken = data.token;
+          done(err);
+        });
+    });
+    it('Should create accommodation successfully', (done) => {
+      router()
+        .post('/api/v1/accommodation')
+        .set('token', authorizedToken)
+        .send(validAccommodation)
+        .end((error, response) => {
+          expect(response.body).to.be.a('object');
+          expect(response.body.status).to.be.equal(201);
+          expect(response.body).to.have.property('message');
+          done(error);
+        });
+    });
+
+    it('Should not create accommodation, invalid inputs', (done) => {
```

Comment:
Improve the description **Should not create accommodation when invalid data is submitted**.

Reply:
Feedback implemented.